tests/tensor/random/test_op.py

```
@@ -22,20 +22,24 @@ def set_aesara_flags():
         yield
 
 
-def test_default_shape_from_params():
+def test_default_supp_shape_from_params():
     with pytest.raises(ValueError, match="^ndim_supp*"):
-        default_shape_from_params(0, (np.array([1, 2]), 0))
+        default_supp_shape_from_params(0, (np.array([1, 2]), 0))
```

Docstring on why this should raise a value error